tools/etcd-top/etcd-top.go

```diff
@@ -0,0 +1,213 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"flag"
+	"fmt"
+	"math"
+	"net/http"
+	"os"
+	"runtime"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/akrennmair/gopcap"
```

Comment:
Are you happy with the README.md's usage of `godep go` for this, or would you prefer the use of relative paths?